docroot/themes/custom/twentyseventeen/sass/components/schedule/_schedule.scss

```diff
@@ -0,0 +1,134 @@
+// Schedule
+//
+// Conference Schedule Layout
+//
+// Markup: schedule.twig
+//
+// Style guide: components.schedule
+
+.program-schedule {
+	
+}
+
+.program-schedule .views-exposed-form {
+	//border: 1px solid #f00;
+	
+}
+
+
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-sort-by,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-sort-order,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-field-experience-level-value,
+.program-schedule .views-exposed-form .form-item.js-form-type-select.form-type-select.form-item-field-session-track-target-id,
+.program-schedule .views-exposed-form .form-actions.js-form-wrapper.form-wrapper {
```

Comment:
<img width="660" alt="broken_tab_display" src="https://user-images.githubusercontent.com/3085730/27342858-56ee47c6-55af-11e7-9ff8-ea3d140ce604.png">
@lkopacz This is what happens when I remove those selectors. The beauty of Views!